Model/Transaction.php

```diff
@@ -200,6 +200,15 @@ protected function buildLineItems($order, $items, $type = 'order') {
             $itemType = $item->getProductType();
             $parentItem = $item->getParentItem();
             $unitPrice = (float) $item->getPrice();
+            $quantity = (int) $item->getQtyOrdered();
+
+            if ($type == 'refund') {
```

Comment:
Any reason not to use `===` strict equality here?

Reply:
@ScottRudiger  I think it's ok to use == here because we're doing a string comparison against a variable with a known type (we explicitly set the value of the string).  It's also consistent with how we do string comparisons in the rest of the code.

We do use === when comparing against true, false, and zero, which could accidentally be  coerced to a different type and cause unexpected results.  

That said, I can't think of a compelling reason not to use === everywhere we currently use ==, but I think that would be more appropriate for a different PR.